src/js/record_audio_worklets.js

```diff
@@ -0,0 +1,20 @@
+import { useDerivedValue } from 'react-native-reanimated';
+
+// Record Audio Worklets
+
+const MAX_SCALE = 1.8;
+
+export function ringScale(scale, substract) {
```

Comment:
should we create a some unit tests for this code? it could be done in a follow up

Reply:
We already have Jest available right? I think the test should come along with the PR.

Reply:
yeah we have jest installed, it's currently only running on clojurescript files so it needs a quick config update to also run on the js files. 
I can set this up and work with Brian on it. I agree going forward the test should come with it but since it's not available just yet perhaps we should not block this p.r from going in,

Reply:
Sure thing, once we have that set up would be great to add unit tests for it, let me know and I can add it on a follow up PR this time :)

Reply:
Makes sense guys, if it's not yet set up it's fine to not to test now.